package.json

```diff
@@ -5,6 +5,11 @@
   "repository": "git@github.com:jean-smaug/koa-twig.git",
   "author": "Maxime BLANC <maximeblanc.dev@gmail.com>",
   "license": "MIT",
+  "keywords": ["koa", "twig"],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jean-smaug/koa-twig.git"
+  },
```

Comment:
Didn't see your PR.
I pushed to update the `repository` key
Can you rebase and remove what I've done in order to keep your work